.env.sample

```diff
@@ -77,3 +78,8 @@ OP_BATCHER_KEY=0xb8971409b424ef3c5bc0ca867b94d81b7fdb2dcbe5625b0906d332359b09c04
 ## `relayer` uses for messaging from L2 to L1
 OP_RELAYER_ADDR=0x6a9a1914b1080C0C0409565A1536B338fb1E51De
 OP_RELAYER_KEY=0x2033aa4d6ddf71c0c486aa722e9eb10aa760a2cde2f38d505a229ed8ebc659ed
+
+## message relayer
```

Comment:
the name should follows as others and please add the address as well, even not using.
```
MR_PROVER_ADDR=0x...
MR_PROVER_KEY=0x...
MR_FINALIZER_ADDR=0x..
MR_FINALIZER_KEY=0x..
```